pandas/core/arrays/categorical.py

```diff
@@ -323,7 +323,7 @@ class Categorical(ExtensionArray, PandasObject):
     # ops, which raise
     __array_priority__ = 1000
     _dtype = CategoricalDtype(ordered=False)
-    _deprecations = frozenset(['labels'])
+    _deprecations = frozenset(['labels', 'tolist'])
```

Comment:
add a note whereever you are doing this that tolist is not actually deprecated, just suppressed in the ``__dir__``. We *could* also make an attribute for this, but prob not worth it